### ISSUE TITLE
Error when creating memory.

### ISSUE THREAD

**Issue by uskudarli** [enshkn/BOUN-SWE-573-SPRING-23#70]
I got this error after entering a memory

time was decade starting at 1970

```bash

Django Version: 4.1.7
Python Version: 3.10.10
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'base.apps.BaseConfig',
 'rest_framework',
 'ckeditor']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/code/base/views.py", line 175, in createMemory
    time_end_obj = datetime.strptime(time_end, '%Y-%m-%d %H:%M')
  File "/usr/local/lib/python3.10/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.10/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %

Exception Type: ValueError at /create-memory/
Exception Value: time data '-12-31 23:59' does not match format '%Y-%m-%d %H:%M'

```

**Comment by enshkn** [comment]
the problem is here; 
base/templates/base/memory_form.html , line: 90-105 in the script section: 

If the user has selected 'interval' as 'time_type' and leaves the time end field empty, the application will be updated to warn them to fill it in. The following script will be added to the relevant place.

if ($('#id_time_type').val() === 'interval' && !timeEnd.isValid()) {
        // Display an error message or perform necessary action
        alert('Please select a valid end time.');
        return false;
      }

if ($('#id_time_type').val() === 'interval' && $('#time_end').val().trim() === ''') {
        alert('Please fill in the time end field.');
        return false;
      }

also the duplicate code under this code will be deleted.

**Comment by enshkn** [comment]
done and closed with the commit 42b569dacb627916e656d72a1052d2e50b1b56c9
